Doc/library/typing.rst

```diff
@@ -1422,19 +1413,15 @@ These can be used as types in annotations. They all support subscription using
         # ``Annotated[list[tuple[int, int]], MaxLen(10)]``:
         type V = Vec[int]
 
-   * ``Annotated`` cannot be used with an unpacked :class:`TypeVarTuple`::
-
-        type Variadic[*Ts] = Annotated[*Ts, Ann1]  # NOT valid
+   ``Annotated`` cannot be used with an unpacked :class:`TypeVarTuple`::
 
-     This would be equivalent to::
```

Comment:
I like the "This would be equivalent to" but I guess we can collapse it onto the same line as you did.